assets/css/less/grids.less

```diff
@@ -54,6 +58,16 @@
 	grid-column: span 11/span 11;
 }
 
+.wpbdp-grid-mixin-loop (@index) when (@index > 0) {
```

Comment:
This is being used generate grids of different dimensions eg `wpbdp-col-4` `wpbdp-col-8` It reduces the less code needed to generate this

Reply:
I see. The Less threw me off.

So .wpbdp8 and .wpbdp-col-8 are duplicate classes then?

Reply:
The `.wpbdp8` is used to set columns with a span of 8/8. For `.wpbdp-col-8` it will only set 1 column. Its sort of set up like bootstrap columns where the max column length is 12 and for positioning the two class combinations for different divs would be `.wpbdp-col-8` and `.wpbdp-col-4`

Reply:
I just tested this, and they are the same thing. I went ahead and removed the duplicates.